server/controllers/auth.js

```diff
@@ -114,3 +114,10 @@ exports.logoutUser = asyncHandler(async (req, res, next) => {
 
   res.send("You have successfully logged out");
 });
+
+// @route POST /auth/logout
+// @desc Ping test with logout controller
+// @access Public
+exports.pingLogout = asyncHandler(async (req, res, next) => {
+  res.send(`Server is running. Message received: ${req.body.message}`);
+});
```

Comment:
Already reviewed in the previous PR. Consider using `git rebase` if you don't want this kind of change to show as new changes in the new branch you checked out to. After making a commit in one branch, if you checkout to a new feature branch, you don't want the changes in the old branch to still show up as new changes in the new branch since you've already made a PR on that, you can just do `<in the new branch> git rebase <old_branch>`.

Reply:
Another suggestion is just to change the base in this PR so that we know it's a branch off of something else :) then the PR won't reflect the files changed in the other branch

Reply:
Oh, it's my first time to hear ```git rebase```. Thanks for your feedback. Applied.